src/SolidStateDetector/DetectorGeometries.jl

```diff
@@ -55,6 +55,22 @@ function yaml2json(directory::String)# or filename
         end
     end
 end
+
+function json2yaml_convert(filename::String)
+    data = JSON.parsefile(filename, dicttype = DataStructures.OrderedDict)
+    YAML.write_file(replace(filename, ".json" => ".yaml"), data)
+end
+function json2yaml(directory::String)# or filename
+    if endswith(directory, ".json")
+        json2yaml_convert(directory)
+    else
+        yamlfiles = filter(x->endswith(x,".json"),readdir(directory))
```

Comment:
I made a small mistake here. It should be:
```julia
yamlfiles = filter(x->endswith(x,".json"), joinpath.(directory, readdir(directory)))
```
Same above in line 52.

And for consistency, we should rename `yamlfiles` in the `json2yaml` function to `jsonfiles`.